crates/turborepo-auth/src/lib.rs

```diff
@@ -38,6 +39,10 @@ impl Token {
     pub fn existing(token: String) -> Self {
         Self::Existing(token)
     }
+    pub fn from_file(path: &AbsoluteSystemPathBuf) -> Result<Self, Error> {
+        let token = std::fs::read_to_string(path)?;
```

Comment:
I don't think this is correct, don't we want to be parsing the contents of the file and extracting the token or have we switched to just saving them raw?

Good candidate for a unit test
```suggestion
    pub fn from_file(path: &AbsoluteSystemPath) -> Result<Self, Error> {
        let token = path.read_to_string()?;
```